wrappers/wrapper.rb

```diff
@@ -727,7 +732,6 @@ def check_distances(vrp, unfeasible)
 
         add_unassigned(unfeasible, vrp, service, 'Service cannot be reached within its timewindows') if service_unreachable_within_its_tw
       }
-
       log "Following services marked as infeasible:\n#{unfeasible.group_by{ |u| u[:reason] }.collect{ |g, set| "#{(set.size < 20) ? set.collect{ |s| s[:service_id] }.join(', ') : "#{set.size} services"}\n with reason '#{g}'" }.join("\n")}", level: :debug unless unfeasible.empty?
```

Comment:
Can we get the space back please? It is my favorite space :) 

Jokes aside it is better to see the end of the loop without clutter for rapid comprehension.